### DISCUSSION TITLE
Mobile jumps somewhere earlier in PR after replying

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#86501] [Mobile] [answered]
I'm reviewing https://github.com/cert-manager/website/pull/1279#pullrequestreview-1810957085

I post a reply near https://github.com/cert-manager/website/pull/1279#discussion_r1447407074

And I'm taken way up in the view and then have to scroll back down to where I was, which is really problematic as I kinda have to expand everything to figure out where that was. (I feel like I'm in a maze of twisty passages all alike.)

https://github.com/cert-manager/website/pull/1279#discussion_r1447420269
And it jumped to
![Screenshot_20240110-085644.png](https://github.com/community/community/assets/2119212/9fc0553f-1b38-440c-92d9-d5a5b0ab2c8a)

**Comment by elisealix22** [comment, marked as answer]
I'm sorry you're losing your place in reviewing, @jsoref ! I'm creating some larger test PRs to help us reproduce and we will look into this. Thank you again for reporting.

**Reply by Vahid8478** [reply]
Very good

**Reply by Vahid8478** [reply]
Very good

**Reply by Vahid8478** [reply]
> Very good 

W0w

**Comment by Vahid8478** [comment]
> I'm reviewing https://github.com/cert-manager/website/pull/1279#pullrequestreview-1810957085
> 
> I post a reply near https://github.com/cert-manager/website/pull/1279#discussion_r1447407074
> 
> And I'm taken way up in the view and then have to scroll back down to where I was, which is really problematic as I kinda have to expand everything to figure out where that was. (I feel like I'm in a maze of twisty passages all alike.)
> 
> https://github.com/cert-manager/website/pull/1279#discussion_r1447420269
> And it jumped to
> ![Screenshot_20240110-085644.png](https://github.com/community/community/assets/2119212/9fc0553f-1b38-440c-92d9-d5a5b0ab2c8a)
> 
>
